src/public/js/views/dashboard/dashboard-controller.js

```diff
@@ -21,7 +22,10 @@ angular.module('rafflesApp.controllers.dashboard', [
   .controller('DashboardCtrl', ['$stateParams', '$sce', 'Raffle', 'Alerts', 'Socket', function($stateParams, $sce, Raffle, Alerts, Socket) {
     var ctrl = this;
 
+    ctrl.editing = false;
     ctrl.raffle = $stateParams.raffle;
+    ctrl.storedRaffle = localStorage.getItem(ctrl.raffle) ? JSON.parse(localStorage.getItem(ctrl.raffle)) : {};
```

Comment:
localStorage can be a bit slow, so it might be worth storing the value of localStorage.getItem() in a variable instead of reading it twice